capirca/lib/nsxt.py

```diff
@@ -13,13 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+#
+# VERSION 1.01
+# Fix the wrong use of notes instead of description, notes are supposed to be used for rule lock/unlock
```

Comment:
This is an important explanation, this would be a clear reason to put the notes into description (possibly in addition to notes).